src/task-dispatcher/container/task.hh

```diff
@@ -16,6 +16,9 @@
 
 #include <task-dispatcher/common/system_info.hh>
 
+// the size of a task in cachelines (= 64B)
+#define TD_FIXED_TASK_SIZE 2

```

Comment:
this is not really public api, right? and does it have to be a macro? a detail:: constexpr/enum value might be a better fit

Reply:
Yeah, was thinking of making it a CMake exposed option at the time but an enum val is just as good for now